ansible_collections/arista/avd/roles/eos_designs/templates/connected_endpoints/port-channel-interfaces.j2

```diff
@@ -76,21 +76,47 @@ port_channel_interfaces:
     mtu: {{ adapter_settings.mtu }}
 {%                         endif %}
 {%                         if adapter_settings.switches | unique | list | length > 1 %}
-{%                             if adapter_settings.port_channel.short_esi is arista.avd.defined %}
-{%                                 if adapter_settings.port_channel.short_esi | lower == 'auto' %}
+{%                             if adapter_settings.port_channel.short_esi is arista.avd.defined and adapter_settings.ethernet_segment.short_esi is not arista.avd.defined %}
+{%                                 do adapter_settings.update({ 'ethernet_segment': { 'short_esi': adapter_settings.port_channel.short_esi }  }) %}
```

Comment:
This update will overwrite everything else defined under `ethernet_segment`. Is that on purpose?
I would prefer to set a `short_esi` variable using the `arista.avd.default` filter and then reference/overwrite that throughout the template.

Reply:
Have made this change - thanks for the feedback, makes it a lot more readable.